dataland-frontend/tests/e2e/fixtures/frameworks/sfdr/SfdrPreparedFixtures.ts

```diff
@@ -1,47 +1,44 @@
 import { generateFixtureDataset } from '@e2e/fixtures/FixtureUtils';
 import { type SfdrData, YesNo } from '@clients/backend';
-import { generateSfdrData, generateSfdrFixtures } from './SfdrDataFixtures';
+import { generateSfdrFixtures } from './SfdrDataFixtures';
 import { type FixtureData } from '@sharedUtils/Fixtures';
 
-type generatorFunction = (input: FixtureData<SfdrData>) => FixtureData<SfdrData>;
-
 /**
  * Generates SFDR prepared fixtures by generating random SFDR datasets and afterward manipulating some fields
  * via manipulator-functions to set specific values for those fields.
  * @returns the prepared fixtures
  */
 export function generateSfdrPreparedFixtures(): Array<FixtureData<SfdrData>> {
-  const manipulatorFunctions: Array<generatorFunction> = [
-    manipulateFixtureForTwoSfdrDataSetsInDifferentYears,
-    manipulateFixtureForOneFilledSubcategory,
-    generateFixtureWithBrokenFileReference,
-    generateFixtureWithIncompleteReferencedReport,
-  ];
-  const preparedFixturesBeforeManipulation = generateFixtureDataset<SfdrData>(
-    generateSfdrData,
-    manipulatorFunctions.length
-  );
   const preparedFixtures = [];
-  for (let i = 0; i < manipulatorFunctions.length; i++) {
-    preparedFixtures.push(manipulatorFunctions[i](preparedFixturesBeforeManipulation[i]));
-  }
 
+  preparedFixtures.push(manipulateFixtureForTwoSfdrDataSetsInDifferentYears(generateSfdrDataWithoutNulls()));
```

Comment:
Das fixture aus `manipulateFixtureForTwoSfdrDataSetsInDifferentYears` wird in keinem test genutzt und kann daher gänzlich entfernt werden.